src/main/java/com/google/sps/data/ExtendedTask.java

```diff
@@ -30,11 +31,24 @@
   @Key
   private long duration;
 
+  /**
+   * @param task represented task
+   * @param duration duration of the task in milliseconds
+   */
   public ExtendedTask(Task task, Long duration) {
     this.task = task;
     this.duration = duration;
   }
 
+  /**
+   * @param task represented task
+   * @param minutes duration of the task in minutes
+   */
+  public ExtendedTask(Task task, String minutes) {
```

Comment:
This is a bit strange. Especially since now there are two constructors, that take duration in different time units.
I think conversion to miliseconds should happen at the call place and `ExtendedTask` should have just 1 constructor.